network/internal/p2pfixtures/fixtures.go

```diff
@@ -334,10 +342,16 @@ func EnsureNoStreamCreation(t *testing.T, ctx context.Context, from []p2p.LibP2P
 				// should not happen, unless the test is misconfigured.
 				require.Fail(t, "node is in both from and to lists")
 			}
-			// stream creation should fail
-			_, err := this.CreateStream(ctx, other.Host().ID())
-			require.Error(t, err)
-			require.True(t, flownet.IsPeerUnreachableError(err))
+			// we intentionally do not check the error here, with libp2p v0.24 connection gating at the "InterceptSecured" level
+			// does not cause the nodes to complain about the connection being rejected at the dialer side.
+			// Hence, we instead check for any trace of the connection being established in the receiver side.
+			otherId := other.Host().ID()
+			thisId := this.Host().ID()
+			_, err := this.CreateStream(ctx, otherId)
```

Comment:
```suggestion
			_, _ := this.CreateStream(ctx, otherId)
```

Reply:
We still need this for external error checking when test needs: 

https://github.com/onflow/flow-go/blob/f20b8c85872bd07097fefde1fadb07bd5ef3ca07/network/internal/p2pfixtures/fixtures.go#L358